packages/dashmate/docs/config/index.md

```
@@ -179,3 +179,52 @@ dashmate config get <option>
 ```
+
+## Running Dashmate commands concurrently
+
+Dashmate keeps all configuration in a single `config.json` inside its home directory
+(`~/.dashmate` by default), and a command that changes configuration rewrites that whole
+file when it exits.
+
+To keep two overlapping commands from silently reverting each other, a command refuses to
+save when `config.json` changed on disk after that command loaded it:
+
+```
```

markdownlint-cli2 (0.23.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)